app/grants/views.py

```diff
@@ -1864,6 +1864,15 @@ def grant_new(request):
 
         grant = Grant.objects.create(**grant_kwargs)
 
+        try:
+            # record to notion for sybil-hunters
+            save_grant_to_notion(title, grant.url)
+        except:
+            # delete the newly created grant and error if we can't save to notion
+            grant.delete()
```

Comment:
so if notion integration field fails this will delete the grant?? if that the case I don't feel super comfortable with the approach.

Reply:
I've altered the process to send an email on failures instead of blocking the save 👍 

Resolved in 351055f

Reply:
13135340